CHANGELOG.md

```diff
@@ -94,8 +95,17 @@ variable "db_host_endpoint" {
   status_update_queue_message_retention_time_seconds   = 777600
```

Comment:
Do we? It's an optional variable, and it doesn't look like Cumulus has added it.

Reply:
Yes, I think i would still add it with its default value to show users that the options is available and can be overridden.

Reply:
Added to side branch as a line reminding how to set TF variables. I would prefer to avoid adding it to this copy/paste section as that will mean we can never reliably change our default.